ModelTraining/tests/test_run_inference.py

```
@@ -85,6 +87,126 @@ def _make_huggingface_model_mock(num_local: int = 10) -> MagicMock:
+    print(f"  Global prediction: {result['global_prediction_label']}")
+    print(f"  Global confidence: {result['global_confidence']:.4f}")
+    print(f"  Probabilities: {result['probabilities']}")
+
+
+def _print_huggingface_result(result: Dict, label: str = "") -> None:
+    """Print HuggingFace inference results for debugging."""
+    prefix = f"HuggingFace inference results{f' on {label}' if label else ''}:"
+    print(f"\n{prefix}")
+    print(f"  Global prediction: {result['global_prediction_label']}")
+    print(f"  Global confidence: {result['global_confidence']:.4f}")
+    print(f"  Probabilities:")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)